argo-rollouts/install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+kubectl create namespace argo-rollouts
+kubectl apply -n argo-rollouts -f https://github.com/argoproj/argo-rollouts/releases/latest/download/install.yaml
```

Comment:
This uses the "latest" release yaml but below you've got a specific version in the manifest.yaml. Would it be best to pin this to a specified version?

Reply:
Thanks, @kaihoffman Makes sense. Have updated the links